packages/mobile/test/schemas.ts

```diff
@@ -186,6 +186,30 @@ export const v4Schema = {
     verificationStatus: 0,
     hasSeenVerificationNux: false,
   },
+  account: {
```

Comment:
This is okay but most of these values are duplicating what was already set for `account` above in v0Schema

Reply:
I wanted to remove `paymentRequests` from the schema, what approach would be better?